Gemfile

```diff
@@ -13,7 +13,7 @@ group :jekyll_plugins do
   gem 'jekyll-asset-pipeline', git: 'https://github.com/crdschurch/jekyll-asset-pipeline', tag: '0.0.2'
   # gem 'jekyll-asset-pipeline', path: File.expand_path('../jekyll-asset-pipeline', __dir__)
   gem "jekyll-contentful", git: 'https://github.com/crdschurch/jekyll-contentful.git', tag: '1.2.1'
```

Comment:
same issue as crds-media, doesn't this need to be 1.3.0?

Reply:
@j-brammer I didn't update Jekyll Contentful for these projects yet. I figured we could do that separately if/when you're ready to do so. Its not critical to the work I'm doing for briantome.com

Reply:
Alrighty